src/sentry/static/sentry/app/types/echarts.tsx

```diff
@@ -19,6 +19,8 @@ export type Series = {
     opacity: number;
   };
   lineStyle?: EChartOption.LineStyle;
+  stack?: string;
+  z?: number;
```

Comment:
please what does z means?

Reply:
Yeah, something like that in echarts world:

> z value of all graphical elements in Line, which controls order of drawing graphical components. Components with smaller z values may be overwritten by those with larger z values.

https://echarts.apache.org/en/option.html#series-line.z